test/test_against_images_refs.py

```diff
@@ -87,6 +88,10 @@ def test_images_ref(tmp_path, monkeypatch, tc):
         ref_manifest = yaml.safe_load(fp)
         normalize_rpm_refs(ref_manifest)
 
+    # hack
+    ref_manifest = json.loads(
```

Comment:
I feel very uneasy about - it's very broad, all UUIDs will be removed, we have some that we care about like part_type (this will work for them as there is a upper/lower case difference here but relying on this seems risky) - I would really like to explore options and not just yet merge this.

Reply:
I opened https://github.com/osbuild/otk/pull/212 with a slightly more targeted version (also hacky but in different ways).

Reply:
Resolve, will be discussed separately.